src/SimplePhotometryModel.cc

```diff
@@ -1,4 +1,5 @@
 #include <iostream>
+#include <math.h>
```

Comment:
c++ code should always use `#include <cmath>` instead.  That may mean you need to add some `std::` in front of math functions, but you should do that anyway.